firefox-settings/pure.js

```diff
@@ -155,3 +155,37 @@ pref("browser.safebrowsing.provider.mozilla.pver", 0, locked);
 pref("browser.safebrowsing.provider.mozilla.reportURL", "", locked);
 pref("browser.safebrowsing.provider.mozilla.updateURL", "", locked);
 pref("browser.safebrowsing.reportPhishURL", "", locked);
+
+pref("browser.contentblocking.category", "strict"); // https://blog.mozilla.org/security/2021/02/23/total-cookie-protection/
+pref("browser.download.autohideButton", false, locked); //do not hide download button automatically
+pref("browser.download.start_downloads_in_tmp_dir", true, locked); //start downloads in temp
+pref("browser.helperApps.deleteTempFileOnExit", true); // https://bugzilla.mozilla.org/show_bug.cgi?id=238789#c19
+pref("browser.tabs.unloadOnLowMemory", true, locked); // Unload unused tabs
+pref("browser.uitour.enabled", false); // https://trac.torproject.org/projects/tor/ticket/19047#comment:3
+
+pref("content.notify.interval", 100000); // page reflow timer, lower redrawn rendering timer, increases responsiveness but increase total load time
+pref("dom.security.https_only_mode", true, locked); // only allow https in all windows, including private browsing
+pref("network.auth.subresource-http-auth-allow", 1, locked); // block HTTP authentication credential dialogs
+pref("network.cookie.sameSite.noneRequiresSecure", true); // https://hacks.mozilla.org/2020/08/changes-to-samesite-cookie-behavior/
+pref("network.dns.disablePrefetch", true); // https://developer.mozilla.org/docs/Web/HTTP/Headers/X-DNS-Prefetch-Control
+pref("network.dns.disablePrefetchFromHTTPS", true); // https://developer.mozilla.org/docs/Web/HTTP/Headers/X-DNS-Prefetch-Control
+
+pref("network.dnsCacheExpiration", 3600); // Time DNS entries are cached in seconds.
+pref("network.http.max-connections", 1800); //Maximum number of any type of connections per server
+pref("network.http.max-persistent-connections-per-server", 10); //Maximum number of keep-alive type connections per server
+pref("network.http.max-urgent-start-excessive-connections-per-host", 5); //Number of connections that we can open beyond the standard parallelism limit defined by max-persistent-connections-per-server/-proxy to handle urgent-start marked requests
+pref("network.http.pacing.requests.enabled", false); //Disable pacing requests
+
+pref("network.predictor.enable-prefetch", false) // Disable speculative website loading
+pref("network.predictor.enabled", false); // Disable speculative website loading
+pref("network.prefetch-next", false); // https://developer.mozilla.org/en-US/docs/Glossary/Prefetch
+pref("network.ssl_tokens_cache_capacity", 10240); // Increase TLS token caching
+pref("nglayout.initialpaint.delay", 0); // How long firefox will wait before rendering page
+pref("nglayout.initialpaint.delay_in_oopif", 0); // How long firefox will wait before rendering page
+pref("privacy.globalprivacycontrol.enabled", true); // https://support.mozilla.org/1/firefox/126.0/Linux/en-US/global-privacy-control
```

Comment:
I'm not sure if we need it. The GPC looks more like a marker for users. It also doesn't work for all regions, probably few people need it when there is already built-in protection against fingerprinting.